pkg/kubelet/server/metrics/metrics.go

```diff
@@ -64,6 +64,18 @@ var (
 		},
 		[]string{"method", "path", "server_type", "long_running"},
 	)
+	// VolumeStatCalDuration tracks the duration in seconds to calculate volume stats.
+	// this metric is mainly for comparison between fsquota monitoring and `du` for disk usage.
+	VolumeStatCalDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      kubeletSubsystem,
+			Name:           "volume_metric_collection_duration_seconds",
+			Help:           "Duration in seconds to calculate volume stats",
+			Buckets:        metrics.DefBuckets,
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"volume_type"},
```

Comment:
"du" isn't really a volume type.  Maybe "metric_source", or something along those lines?

Reply:
will update. `metric_source` is more accurate.